app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -30,6 +30,10 @@ def can_scale_manually?
         false
       end
 
+      def Default.supports?(as)
```

Comment:
Unused method argument - `as`. If it's necessary, use `_` or `_as` as an argument name to indicate that it won't be used. You can also write as `supports?(*)` if you want the method to accept any arguments but don't care about them.<br>Use `self.supports?` instead of `Default.supports?`.